test/windows-bootstrap.el

```diff
@@ -41,6 +40,16 @@
         pkgs)
 
   (add-hook 'kill-emacs-hook
-            `(lambda () (delete-directory ,user-emacs-directory t))))
+            `(lambda ()
+               ;; NOTE: If you are testing this on your own PC, then you
+               ;; might want to uncomment the line below to do the clean up
+               ;; after the test. (This will take more time to complete the
+               ;; CI process)
+               ;;
+               ;; Please leave it comment in CI so we don't accidentally install
+               ;; depedencies twice.
+               ;;(delete-directory ,user-emacs-directory t)
```

Comment:
Instead of this, we could check a ENV var that we would always set on CI, like `EMACS-LSP-CI=1`, so local tests would just work, WDYT?

Reply:
Good idea. Added variable `emacs-lsp-ci`.